reana_commons/config.py

```diff
@@ -413,6 +413,9 @@ def default_workspace():
 account.
 """
 
+SLURM_PARTITIONS = ["inf-short", "inf-long", "photon", "qcd", "qcd-backfill"]
```

Comment:
The partition names are very CERN-centric, so it may confuse other REANA deployments who might like to introduce Slurm bridge. In view of future generic applicability, perhaps we should label these `SLURMCERN_...` so that it's clear that these will be force-checked only by `SlurmJobManagerCERN`?

Alternatively, we can accept any string that user specifies, so that we are more future-proof. So the check would lead to just a warning and a "continuable error" kind of situation, not a hard "fatal error" that would prevent job submission.